force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -512,6 +512,13 @@
         <shortDescription>Update</shortDescription>
         <value>Update</value>
     </labels>
+    <labels>
+        <fullName>Edited</fullName>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Edited</shortDescription>
+        <value>Edited</value>
```

Comment:
@jessicahaas1 this is the only label here that needs your approval. It's for rows in BSDT that have been saved, but then the user has edited. Let me know if you need further detail before approving the PR. Thank you!

Reply:
<shortDescription>Indicates an entry that has been saved, then edited.</shortDescription>

Reply:
We usually don't use the description field here. It doesn't actually show up anywhere except in setup>labels, where the Admin has no need to actually be. Only the word "Edited" shows up in the UI, and this could be reused elsewhere.

Reply:
I see- in that case, looks great

Reply:
@bethbrains - what do you think about putting the description in the hover of the icon like we have for the error?

Reply:
I think "Edited" is enough.